tests/Propel/Tests/Generator/Reverse/MysqlSchemaParserTest.php

```diff
@@ -8,42 +8,85 @@
 
 namespace Propel\Tests\Generator\Reverse;
 
-use Propel\Generator\Config\QuickGeneratorConfig;
-use Propel\Generator\Model\Database;
-use Propel\Generator\Platform\DefaultPlatform;
+use PDO;
 use Propel\Generator\Reverse\MysqlSchemaParser;
-use Propel\Runtime\Propel;
-use Propel\Tests\TestCaseFixturesDatabase;
+use Propel\Tests\Bookstore\Map\BookTableMap;
 
 /**
  * Tests for Mysql database schema parser.
  *
  * @author William Durand
  *
  * @group database
+ * @group mysql
  */
-class MysqlSchemaParserTest extends TestCaseFixturesDatabase
+class MysqlSchemaParserTest extends AbstractSchemaParserTest
 {
+    /**
+     * @return string
+     */
+    protected function getDriverName(): string
+    {
+        return 'mysql';
+    }
+
+    /**
+     * @return string
+     */
+    protected function getSchemaParserClass(): string
+    {
+        return MysqlSchemaParser::class;
+    }
+
+    /**
+     * @return void
+     */
+    protected function setUp(): void
+    {
+        parent::setUp();
+    }
+
     /**
      * @return void
      */
-    public function testParse()
+    public function testParseImportsAllTables()
     {
-        $this->markTestSkipped('Skipped as we now use one database for the whole test suite');
+        $query = <<< EOT
+SELECT table_name
+FROM INFORMATION_SCHEMA.TABLES
+WHERE table_schema=DATABASE()
+AND table_name NOT LIKE 'propel_migration'
+EOT;
+        $expectedTableNames = $this->con->query($query)->fetchAll(PDO::FETCH_COLUMN, 0);
 
-        $parser = new MysqlSchemaParser(Propel::getServiceContainer()->getConnection('reverse-bookstore'));
-        $parser->setGeneratorConfig(new QuickGeneratorConfig());
+        $importedTables = $this->parsedDatabase->getTables();
+        $importedTableNames = array_map(function ($t) {
```

Comment:
Small suggestion. To make code more clear better to not use abbreviations, so I suggest rename `$t` to `$table`.

Reply:
sure